.github/workflows/main.yaml

```diff
@@ -0,0 +1,38 @@
+name: Build and Deploy to ECR
+on:
+  push:
+    branches: [ "develop" ]
+jobs:
+  build-and-push-ecr:
+    runs-on: ubuntu-latest
+    environment: development
+    env:
+      ARKCASE_VERSION: 2021.03-RC11
+    steps:
+      - uses: actions/checkout@v2
+      - name: configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_KEY }}
+          aws-region: ${{ secrets.AWS_REGION }}
+      - name: download artifacts
+        run: ./getArtifact.sh
+        env:
+          ARKCASE_VERSION: 2021.03-RC11
```

Comment:
Isn't ARKCASE_VERSION define on line 10?

Reply:
@naveen-armedia , do we need `ARKCASE_VERSION` at line 10 and also at line 22?  Or, is the `env` in the steps is different from the top-level `env`?